conf/modules.config

```diff
@@ -589,6 +589,40 @@ process {
         ]
     }
 
+    //
+    //  BEDTOOLS_COVERAGE
+    //
+    withName: SAMTOOLS_VIEW_GENOME {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.args = '-H --output-fmt sam'
+        ext.prefix = "genome"
+        ext.args2 = "&& cat genome.sam |  grep '@SQ' | sed 's#@SQ\tSN:\\|LN:##g' > genome.txt && mv genome.txt genome.sam"
+        publishDir = [
+            enabled: false
+        ]
+    }
```

Comment:
Clever hack but probably safer as a standalone local module for this particular step